apps/emails-and-messages/src/lib/channel-assignment/ui/universal-channels-section.tsx

```diff
@@ -0,0 +1,125 @@
+import { BoxWithBorder } from "../../../components/box-with-border";
+import { Box, Button, ProductsIcons, Switch, TableEditIcon, Text } from "@saleor/macaw-ui/next";
+import { defaultPadding } from "../../../components/ui-defaults";
+import { trpcClient } from "../../../modules/trpc/trpc-client";
+import {
+  SmtpUpdateChannels,
+  smtpUpdateChannelsSchema,
+} from "../../../modules/smtp/configuration/smtp-config-input-schema";
+import { Controller, useForm } from "react-hook-form";
+import { BoxFooter } from "../../../components/box-footer";
+import { SectionWithDescription } from "../../../components/section-with-description";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Multiselect } from "../../../components/react-hook-form-macaw/Multiselect";
+import { AssignedChannelsMessage } from "./assigned-channels-message";
+import { ChannelConfiguration } from "../channel-configuration-schema";
+
+interface UniversalChannelsSectionProps {
+  configurationId: string;
+  channelConfiguration: ChannelConfiguration;
+  mutate: any; // TODO: narrow the type
```

Comment:
component doesnt need to use graphql/react-query naming. This is a simple "onSubmit" callback 

```
onSubmit(data: FormData): void
```